pages/signUp.js

```diff
@@ -0,0 +1,48 @@
+import Head from 'next/head';
```

Comment:
I noticed that the login.js and login.css files are included in this PR. However, I see that they are not on the same branch. Just wanted to double-check if this was intentional or if there might have been a mix-up

Reply:
Thanks for pointing that out. It was not intentional and I'll fix it.